src/pages/polska-spolecznosc.html

```diff
@@ -90,11 +90,13 @@ <h3><strong>Fundatorzy</strong></h3>
             <img src="images/members/karolina-sarek.jpg" class="person__img" alt="">
             <div class="person__name">Karolina Sarek</div>
 
-            <div class="person__desc">
-              Odpowiada za strategię i wsparcie organizacji. Interesuje ją
-              kognitywistyka, racjonalność i zastosowanie nowoczesnych
-              technologii do czynienia dobra. Aktywistka Otwartych Klatek.
-              Czyta i podróżuje.
+            <div class="person__desc person__desc__small">
```

Comment:
refactoring to use the class & merging.

Reply:
Hmm... after this refactoring the fix doesn't work, at least on firefox. The % way of specifying font size works so that you specify the % of the *parent* element font size. That's why I created the inner <span> block, so that it's size was 90% of the yellow div, not 90% of the font on the whole page...

Reply:
you are right. I've thought it's a glitch on my zoom-level. Anyway, reset as the original PR:
https://github.com/eapoland/landing-page/commit/ffdb72faa4f347e5a8702b8b3b75c648ab8b21dc